token-metadata/0xbC396689893D065F41bc2C6EcbeE5e0085233447/metadata.json

```diff
@@ -4,5 +4,23 @@
     "address": "0xbC396689893D065F41bc2C6EcbeE5e0085233447",
     "decimals": 18,
     "dharmaVerificationStatus": "VERIFIED",
-    "tags": ["yield-farming"]
+    "tags": ["synthetics","amm","defi","perpetuals"],
```

Comment:
per comment above, please remove `perpetuals` from the tags list

Reply:
okay, thanks for the correction